Mathlib/LinearAlgebra/AffineSpace/AffineMap.lean

```diff
@@ -568,6 +568,10 @@ theorem lineMap_vsub_lineMap (p₁ p₂ p₃ p₄ : P1) (c : k) :
 @[simp] lemma lineMap_lineMap_right (p₀ p₁ : P1) (c d : k) :
     lineMap p₀ (lineMap p₀ p₁ c) d = lineMap p₀ p₁ (d * c) := by simp [lineMap_apply, mul_smul]
 
+@[simp] lemma lineMap_lineMap_left (p₀ p₁ : P1) (c d : k) :
+    lineMap (lineMap p₀ p₁ c) p₁ d = lineMap p₀ p₁ (1 - (1 - d) * (1 - c)) := by
```

Comment:
Or maybe `c + d - d * c`

Reply:
That's worse in that it would make this lemma not be affine: `c` and `d` appear twice in the RHS of your lemma, which is more than on the LHS